### ISSUE TITLE
`command` rule should use config "exec"

### ISSUE THREAD

**Issue by cristifalcas** [ash2k/bazel-tools#27]
Any reason for setting the `cfg` parameter for `command` rule as "target", instead of "exec"?
[Here](https://github.com/ash2k/bazel-tools/blob/60261202f33bd2537264040b7c741768c0ed8b3b/multirun/def.bzl#L233)

The commands are run on the "exec" platform.
I don't see a `command` rule as being the target used at runtime, but mostly to run other tools.
For example we use it to push containers, or `kubectl` commands, both of which we need to run on `exec` platform

**Comment by ash2k** [comment]
Thanks for reporting. Looks like you are correct. Docs here https://bazel.build/extending/rules#configurations. PRs welcome :) If you could check other rules too, that'd be much appreciated!

**Comment by cristifalcas** [comment]
Hi @ash2k . 

My original issue was a mistake: the rule is correct on having the config set to `target`, and on my side in the end the problem was how I was configuring `rules_k8s`.

For extra context, I was setting the "target" `kubectl` instead of the "exec" one to `rules_k8s` and the `rules_k8s` targets were all executed with the `command` rule.
After properly debugging this I found the real issue.

I made a PR to move from the old, deprecated `host` config to the recommended `exec` one.
